packages/tokens/build.js

```diff
@@ -1,6 +1,42 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const StyleDictionary = require('style-dictionary')
 
+/**
+ * Utils
+ */
```

Comment:
Moved these utils to the top since some of them need to be referenced by the filters/formats